src/templates/Article.jsx

```diff
@@ -2,14 +2,50 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Helmet from 'react-helmet';
+import cheerio from 'cheerio';
```

Comment:
It would be crazy to ship a 400k lib to each client just to populate meta tags.

This could be done during the build with a plugin, like we have with the forum emoji [here](https://github.com/freeCodeCamp/guides/tree/master/plugins)

Reply:
I have not worked with Gatsby in the past but I assumed `npm run build` would just generate all the html files needed to build the final website.

Therefore, the final html files would not have any reference to cheerio, right?

If I am wrong and it's not the case we can think of a way to effectively traverse the html content of articles.

Reply:
Only just seeing this sorry. In answer to your question, yes gatsby generates static HTML files for each page, but at the client, it is just a react app.